region/crateria/central.json

```diff
@@ -111,6 +115,7 @@
           "name": "Ship",
           "nodeType": "door",
           "nodeSubType": "elevator",
+          "nodeAddress": "",
```

Comment:
Was that forgotten or missing? Do we want a FIXME note?

Reply:
Oh, it's the ship. Does it need an address?

Reply:
```suggestion
          "note":  "FIXME If there's a proper memory address to the ship/elevator it was in hiding",
```

(I would remove the nodeAddress line completely for now)